util/stats/stats_unix_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2017 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// +build !windows
+
+package stats
+
+import "testing"
+
+// TestTimerGroup does not work on Windows since its timer is not accurate
```

Comment:
That doesn't sound right, Windows has high resolution timers too. This sounds like a bug in the timer code.

Reply:
Time.Time() returns the same if call consecutively on Win.

Added  Sleep()

Reply:
What version of Windows? It really sounds like we're using the wrong timing API here.